### PR TITLE
Fixate spinner center

### LINKED ISSUE #3473: Center spindle spins on the spinner
The Center spindle spins on the spinner, i feel this shouldnt happen as the center spindle is the part that allows the spinner to actually spin without going all over the place, so it really should stay still

### PR DESCRIPTION
**Pull request by AtomCrafty**
- Closes #3473

Not my work, I merely wrapped the changes proposed by @Hubza into a PR since they seemed to have some issues. Please discuss with them whether it should be merged.

### CODE REVIEW

File: osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSpinner.cs [hunk 1]
```diff
@@ -178,14 +178,11 @@ protected override void UpdateAfterChildren()
...
             base.UpdateAfterChildren();
 
-            circle.Rotation = Disc.Rotation;
             Ticks.Rotation = Disc.Rotation;
             spmCounter.SetRotation(Disc.RotationAbsolute);
 
             float relativeCircleScale = Spinner.Scale * circle.DrawHeight / mainContainer.DrawHeight;
             Disc.ScaleTo(relativeCircleScale + (1 - relativeCircleScale) * Progress, 200, Easing.OutQuint);
-
-            symbol.RotateTo(Disc.Rotation / 2, 500, Easing.OutQuint);
```
Comment (peppy) [inline osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSpinner.cs]: this removes the current extra visual cue that the object is a spinner, making it even more like a normal hitcircle. not sure if this is a direction we want to go.

Comment (AtomCrafty) [reply osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSpinner.cs]: I don't really have an opinion on this. @Hubza

Comment (Aergwyn) [reply osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSpinner.cs]: Especially with HD where only the center is visible and thus you don't even know if you spin.

**Comment by LittleEndu** [comment]
#3473 is a feature. That being said, spinners were supposed to get a redesign sometime in #1822

**Comment by AtomCrafty** [comment]
In that case this PR and the referenced issue should be closed I guess?

**Comment by peppy** [comment]
Leave open for now, but I believe a bit more thought needs to be put into this.

**Comment by peppy** [comment]
Closing this one for now. I think we will see a spinner redesign in the coming months.

**Pull request closed without merging**